index.bs

```diff
@@ -178,8 +177,8 @@ This specification relies on several other underlying specifications.
 
 : HTML
 :: The concepts of <a link-for='web'>current settings object</a>, <a link-for='web'>origin</a>,
-    <a link-for='web'>opaque origin</a>, <a>relaxing the same-origin restriction</a>, and the <a>Navigator</a> interface are
-    defined in [[!HTML51]].
+    <a link-for='web'>opaque origin</a>, <a link-for='web'>is a registrable domain suffix of or is equal
```

Comment:
fyi/fwiw, wrt "mistake in anchors block" mentioned by @jyasskin (which may have been mine): see #336